content/sigs/pipeline-authoring/index.adoc

```diff
@@ -99,3 +94,29 @@ best practices, and examples.
   basic or starter `Jenkinsfile`s for common patterns that can be used
   as jumping-off points by new users.
 
+=== Communications
+The target aduience are anyone who are interested to join the effort of improving and/or curating the experience of writing Jenkins Pipelines.
+As such, anyone with interest to either (or both) join our SIG meetings or our Gitter channel are welcome to participate!
+
+==== Gitter channel (gitter: jenkinsci/pipeline-authoring-sig)
+While any and all with questions wrt. writing a Jenkins Pipeline are welcome in our Gitter channel, we ask that the question has been researched _some_ before it's brought to us.
+Indeed, there are multiple blog posts and guides on the internet discussing how both to _ask_ a question in a constructive manner, and how to _reply_ constructively.
```

Comment:
IMHO it could be moved to a whatever generic location like https://jenkins.io/chat/ and then referenced from here

Reply:
@oleg-nenashev Awesome suggestion! Only caveat I've got is to say that I feel much less comfortable editing/referencing something that general. I'd rather you (or someone else) copy/pasted my stuff there first, before I reference it in this specific PR's edits and merge it =)

Reply:
@oleg-nenashev 
In the long having a this on the chat page makes sense, but for now this is good.